inventory/sample/group_vars/k8s-cluster/k8s-cluster.yml

```diff
@@ -217,8 +217,8 @@ podsecuritypolicy_enabled: false
 ## See https://github.com/kubernetes-sigs/kubespray/issues/2141
 ## Set this variable to true to get rid of this issue
 volume_cross_zone_attachment: false
-# Add Persistent Volumes Storage Class for corresponding cloud provider ( OpenStack is only supported now )
-persistent_volumes_enabled: false
+# Add Persistent Volumes Storage Class for corresponding cloud provider (supported: in-tree OpenStack, Cinder CSI, AWS EBS CSI)
+persistent_volumes_enabled: true
```

Comment:
I don't think that we want to change the default value from false to true here.

Reply:
Oh yeah sorry I forgot to switch it back. Thanks for catching that :D